transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollDomainSocketChannelConfig.java

```diff
@@ -58,11 +58,6 @@
 
         return true;
     }
-    @Override
-    public EpollDomainSocketChannelConfig setMaxMessagesPerRead(int maxMessagesPerRead) {
```

Comment:
I'm still not 100 % convinced that we can do this change in 4.1 in terms of API breakage. @trustin WDYT ?